velox/docs/functions/spark/map.rst

```diff
@@ -0,0 +1,34 @@
+===========================
+Map Functions
+===========================
+
+.. function:: getmapvalue(map(K,V), key) -> V
+    :noindex:
+
+    Returns value for given ``key``, or ``NULL`` if the key is not contained in the map.
+
+.. function:: element_at(map(K,V), key) -> V
```

Comment:
What's the difference between getmapvalue and element_at?

Reply:
getmapvalue is invalid. just removed this one.